content/15.development_procedure.md

```diff
@@ -66,6 +82,30 @@ The workflow for code review roughly follows this outline:
 One increasing issue with the code review process is ensuring that changes are reviewed with appropriate urgency; larger pull requests tend to languish without review, as the requirements for review necessarily add burden to the maintainers.
 "Bugfix" changes formally require only one reviewer, whereas the `yt` guidelines suggest that larger changes require review from two different team members.
 
+One of the most pressing bottlenecks with code review is that the time it takes for tests to pass is much longer than the typical time span during with code review takes place.
+This results in a pull request being merged, and then whatever pull request is next to be reviewed has to wait until the tests (now updated with the newly accepted pull request) pass.
```

Comment:
This sentence doesn't resonate with my personal experience as a reviewer; I don't think that merging a PR has any noticeable impact on reviewing other PRs in the following minutes because they will typically have run their jobs already. The actual limitation can be seen in a different situation, I think, where reviewers and authors have to iterate over a PR, as the CI can't go over more than a couple cycles per hour.

Reply:
Hmm, usually when I'm reviewing a PR I want all the tests to remain green, so if I merge one and it causes tests to rerun, we have to wait.